src/setup/serverless-config.go

```diff
@@ -45,42 +46,56 @@ type FunctionPackage struct {
 }
 
 type Event struct {
-	HttpApi HttpApi `yaml:"httpApi"`
+	AWSHttpEvent   AWSHttpEvent `yaml:"httpApi,omitempty"`
+	AzureHttp      bool         `yaml:"http,omitempty"`
+	AzureMethods   []string     `yaml:"methods,omitempty"`
+	AzureAuthLevel string       `yaml:"authLevel,omitempty"`
 }
 
-type HttpApi struct {
+type AWSHttpEvent struct {
 	Path   string `yaml:"path"`
 	Method string `yaml:"method"`
 }
 
 var nonAlphanumericRegex *regexp.Regexp = regexp.MustCompile(`[^a-zA-Z0-9 ]+`)
 
 const (
-	AWS_DEFAULT_REGION = "us-west-1"
-	GCR_DEFAULT_REGION = "us-west1"
+	AWS_DEFAULT_REGION   = "us-west-1"
+	AZURE_DEFAULT_REGION = "Southeast Asia"
```

Comment:
If possible, we should set this to US West region as well (or it's equivalent in Azure) for some consistency across providers. We can take note first and change this once the credentials are switched to the official account.